### ISSUE TITLE
Rename repo to wai-wcag-act-rules 

### ISSUE THREAD

**Issue by SteveALee** [w3c/wcag-act-rules#61]
For consistency with all the other `wai-website` resources this should be renamed to `wai-wcag-act-rules `

NB! this needs to be done in coordination with changes to `wai-website`

**Comment by SteveALee** [comment]
@WilcoFiers do let me know if you get a suitable time to do this.

**Comment by WilcoFiers** [comment]
Why do we need such a long name?

**Comment by SteveALee** [comment]
All the WAI website resources repos have the `wai-` prefix. Except this one. The prefix makes them easy to identify amongst all the W3C repos.

Why is the length a problem :) Happy for you to change the rest but WCAG probably an useful part.

**Comment by WilcoFiers** [comment]
OK, but there's a good deal more that would need to happen:

- Update wai-website
- Update Michael's ACT import script
- Update the repo field in all the rule files
- Update the act-tools template and tests
- Update the ACT TF rule tracker
- Netlify setup? (not sure if that one needs to change)

If you're volunteering to do all of that, go right ahead. Sooner rather than later I'd say. We won't be able to change this quite so easily once we have automated builds set up, which I'm hoping to do in the next few months.

We the repo this name because Michael wants all AG repos to start with "wcag". What you're proposing goes against that. I don't much care either way, but I would ask that we keep things stable. Breaking changes like this are fairly disruptive.

**Comment by SteveALee** [comment]
That's certainly more involved than expected/hoped. I was ready to do the first and last.

For future ref I'm pretty sure that if repos name in front matter are all the same they can be defined in the jekyll config file rather than each file. That said, I've done the same in the DesignGuide. At least global search and replace is fairly easy if ever need s to happen.

**Comment by SteveALee** [comment]
> We gave the repo this name because Michael wants all AG repos to start with "wcag". What you're proposing goes against that. I don't much care either way, but I would ask that we keep things stable. Breaking changes like this are fairly disruptive.

We want all the wai sites to start with `wai-`. what you did went against that! :-P

**Comment by SteveALee** [comment]
Serious though, "ownership" of the repo is getting blurred in a few cases. In retrospect I might have split this into two -one wcag one wai but that's extra burden so probably would not.
